javascript/javascript2/week1/findShorty.js

```diff
@@ -0,0 +1,36 @@
+const danishWords = [
+    "slot",
+    "skolen",
+    "minderig",
+    "venskab",
+    "harmelig",
+    "kransekage",
+    "rengøring",
+    "fyr",
+    "øl",
+    "egentlig",
+  ];
+  
+  const spanishWords = [
+    "hola",
+    "Argentina",
+    "café",
+    "o",
+    "gato",
+    "perro",
+    "y",
+    "empanadas",
+    "asado",
+    "cerveza",
+  ];
+  
+
+function wordsLengths(arr) {  
```

Comment:
You can reduce the code with ES6 
const getShortest = Math.min(...arr.map((el) => el.length));
 return arr.filter((el) => {return el.length === getShortest})